### ISSUE TITLE
Custom operations on faces like Mask detection.

### ISSUE THREAD

**Issue by snehitvaddi** [ageitgey/face_recognition#1127]
### Description
Hey, I am trying to create an application to detect if a person is wearing Mask or not. How to do similar operations on images.

### **My other questions**
1. Is there any way I could get location of faces in an image?
2. How to perform specific operations like implementing face tracking using other modules.

**Comment by Latestion** [comment]
> * Is there any way I could get location of faces in an image?

> * How to perform specific operations like implementing face tracking using other modules.
Yes both of them is possible you can read the docs for some ideas.
